ansible/roles/sonicv2/templates/etc/systemd/system/syncd.j2

```diff
@@ -7,11 +7,13 @@ After=database.service
 User=root
 {% if sonic_hwsku == 'ACS-MSN2700' %}
 ExecStartPre=/etc/init.d/sxdkernel start
+ExecStartPre=/usr/bin/mst start
```

Comment:
Motivation is same as with sxdkernel - keep service self contained and make it take care of needed resources instead of relying on someone else